WooCommerce/Classes/ViewRelated/Blaze/BlazeCampaignList/BlazeCampaignIntroView.swift

```diff
@@ -0,0 +1,120 @@
+import SwiftUI
+
+/// View to display the introduction to the Blaze campaign
+///
+struct BlazeCampaignIntroView: View {
+    /// Scale of the view based on accessibility changes
+    @ScaledMetric private var scale: CGFloat = 1.0
+
+    var body: some View {
+        NavigationView {
+            ScrollView {
+                VStack(alignment: .leading) {
+                    Image(uiImage: .blaze)
+                        .frame(width: 100, height: 100)
```

Comment:
Nit: the numbers should be moved to the `Layout` enum. Also, other things to consider: 
- we should multiply these with the scale to scale the image for accessibility.
- the image should be resizable to change the size of the content when frame changes.

```suggestion
                        .resizable()
                        .frame(width: Layout.logoSize * scale, height: Layout.logoSize * scale)
```

Reply:
Learned something new, thanks!

Reply:
`logoSize` added in https://github.com/woocommerce/woocommerce-ios/pull/10946/commits/d3f014cbd2da6db02e809874a20fc8b2f1db1f34